DEVELOPER_GUIDE.md

```diff
@@ -178,6 +178,12 @@ In order to run the integration tests with a 3 node cluster, run this command:
 ./gradlew :integTest -PnumNodes=3
 ```
 
+Integration tests can be run with remote cluster. For that run the following command and replace host/port/cluster name values with ones for the target cluster:
+
+```
+./gradlew :integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="integTest-0" -Dhttps=false
```

Comment:
shall we include an example with secure cluster (username/password) as well?

Reply:
Sure, that makes sense. Let me update the guide

Reply:
Added instructions for secure cluster